README.md

```diff
@@ -30,6 +30,4 @@ npx @nacelle/create your-project-name
 
 ## How to Contribute
 
-- [Submit a Pull Request](./CONTRIBUTING.md)
-- [Report bugs](./BUG_REPORT.md)
-- [Suggest an enhancement](./SUGGEST_ENHANCEMENT.md)
+Please see our [Contribution Guidelines](./CONTRIBUTING.md) for more information about submitting bug reports, feature requests, and code contributions.
```

Comment:
Wondering if we should move the Bug/Feature request stuff here directly? There's quite a bit in the contributing guide so people might not necessarily navigate all the way there? If not, we should definitely link directly to the bug reports & feature requests sections so people can be taken right to where they need.

Reply:
Great suggestions! Both have been implemented in https://github.com/getnacelle/nacelle-js/pull/47/commits/ded0574526462ea624b3cc66748ced363e1582d8.